test/tm-cases/break_on_newline_excessive_br_tags_in_ul.html

```diff
@@ -3,17 +3,21 @@ <h2>Conteúdo</h2>
 <ul>
 <li>O que é estrutura Co-locada (on premises), o que é estrutura híbrida e o que é estrutura em-nuvem?
 <ul>
-<li>Em Nuvem (cloud based)</li>
+<li>Em Nuvem (cloud based)
+<ul>
```

Comment:
Doing a quick look at this and it seems wrong because the LIs arent closing. But i'm guessing this is intentional? UL should be nested under the LI before it's closed?

Reply:
The [Mozilla docs](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/HTML_text_fundamentals#nesting_lists) on nesting lists give an example where the nested `<ul>` is placed as a child of the `<li>` before closing and I also found this [stackoverflow answer](https://stackoverflow.com/questions/5899337/proper-way-to-make-html-nested-list) that confirms this